### PR TITLE
Add sociomantic-tsunami/swarm project

### PR DESCRIPTION
**Pull request by matthias-wende-sociomantic**
This is a naive attempt to add swarm to the dlang ci, now that it is a D2 project .

### CODE REVIEW

**Comment by Geod24** [comment]
Side note: You might want to take care of [those deprecations](https://buildkite.com/dlang/ci/builds/659#ad1c5faa-e213-4f1e-8943-046a677fbc6b/134-169) before they turn into errors :)

File: buildkite.sh [hunk 1]
```diff
@@ -121,6 +121,7 @@ projects=(
     "vibe-d/vibe.d+libasync-base" # 3m45s
     "dlang/phobos" # 4m50s
     "sociomantic-tsunami/ocean" # 4m49s
+    "sociomantic-tsunami/swarm"
```
Comment (leandro-lucarella-sociomantic) [inline buildkite.sh]: I don't know how important is to keep the time sorting....

File: buildkite/build_project.sh [hunk 2]
```diff
@@ -211,7 +211,9 @@ case "$REPO_FULL_NAME" in
         dub test "--compiler=$DC"
         ;;
 
-    sociomantic-tsunami/ocean|sociomantic-tsunami/turtle)
+    sociomantic-tsunami/ocean | \
+    sociomantic-tsunami/turtle | \
+    sociomantic-tsunami/swarm)
```
Comment (leandro-lucarella-sociomantic) [inline buildkite/build_project.sh]: This could have been done more future proof by using a case like `sociomantic-tsunami/*)`.

Comment (matthias-wende-sociomantic) [reply buildkite/build_project.sh]: I might do that in the next PR, when adding the dhtproto.

**Pull request merged**